### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by charlypa**
Updating vimrc settings in read me

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,5 +17,88 @@ This utility takes config.yaml as input. Configuration for each app is self-cont
 
 ## Contributing
 1.Use vim or InteliJUltimate or pycharm or visualstudio code.
+### Vim
 
+Following are the vimrc setting used to create this utility. Use it if you are fan of Vim.
```
Comment (abose) [inline README.md:22]: IDE/Editor-specific setup.  It is a good candidate to set up as a template for repos here https://github.com/aicore/core.ai/issues/259 .

Comment (abose) [inline README.md:22]: IDE setups are best done in a generic wiki than with individual repo pull requests. Should we discard this PR and move it to a reusable wiki? Can be a part of https://github.com/aicore/core.ai/issues/259

**Review by abose — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (charlypa) [reply README.md:22]: In my opinion, having a vim configuration is would help someone to start with the development. Once we have the traction, we could move it to the right place, as you suggested.

Comment (charlypa) [reply README.md:22]: In my opinion, having a vim configuration is would help someone to start with the development. Once we have the traction, we could move it to the right place, as you suggested.

**Comment by abose** [comment]
Please move the vim rc file to the template. 
https://github.com/aicore/Template-PythonProject

Closing pull request as the upstream template merge will add the feature to this repo.

**Pull request closed without merging**